gradle.properties

```
@@ -1,11 +1,28 @@
-group=me.champeau.jmh
-version=0.7.3-SNAPSHOT
-
-project_description=Integrates the JMH microbenchmarking framework with Gradle, providing conventional ways to setup sources and execute micro-benchmarks with JMH. Also known as the jmh-gradle-plugin.
-project_website=https://github.com/melix/jmh-gradle-plugin
-project_issues=https://github.com/melix/jmh-gradle-plugin/issues
-project_vcs=https://github.com/melix/jmh-gradle-plugin.git
-
 org.gradle.jvmargs=-Xmx4g -Dfile.encoding=UTF-8
 org.gradle.parallel=true
 org.gradle.caching=true
+
+
+GROUP=me.champeau.jmh
+POM_ARTIFACT_ID=jmh-gradle-plugin
+VERSION_NAME=0.7.3-SNAPSHOT
+
+SONATYPE_AUTOMATIC_RELEASE=true
+SONATYPE_HOST=DEFAULT
+RELEASE_SIGNING_ENABLED=true
+
+POM_NAME=Gradle Plugin for JMH
+POM_DESCRIPTION=Integrates the JMH microbenchmarking framework with Gradle, providing conventional ways to setup sources and execute micro-benchmarks with JMH. Also known as the jmh-gradle-plugin.
+POM_URL=https://github.com/melix/jmh-gradle-plugin
+
+POM_LICENSE_NAME=The Apache Software License, Version 2.0
```

```suggestion
POM_LICENSE_NAME=Apache-2.0
```

It's the SPDX identifier (easier to map by downstream license checking tools)